dev-lang/ispc/Manifest

```diff
@@ -1 +1,7 @@
-DIST ispc-1.13.0.tar.gz 19202295 BLAKE2B 5fae81429acbda0d65d33e56609d05fb8b5a7db6dd0af5ff779d1e8954f6a98d1fcb0f7ecf0c1b9813630026557c21c2a70bfc87ed75959a8bd24e06fcd2e351 SHA512 d23c127beb9bb17e1ad2509f9ad51dd660c473458b939a6b64ca7f6319b3c2f25318ea2ceac00e4100a059f3f67fb97b6973f18e3432c82ff9691f2a08d15d5d
+AUX ispc-1.14.0-cmake-gentoo-release.patch 507 BLAKE2B e67be17f40dce55895ce307bd5383ad3f9dc8ec460c7195e23ce344f3a0324c53f9f69b695b6feb70d18da813d5d855d675a00e8dfd3e651aaaebf510de1138b SHA512 bdfd7d3f023ace4a0209201e05b7528e50cff644794bc48b5160d8051c2d70dad6690e509ca69102b08843ebb5fd7a76241954b22a0bb1f3c4714708df405d65
```

Comment:
Got you! You need to [enable](https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf#thin-manifests) thin-manifests on your testing overlay because the AUX/EBUILD/MISC lines are not necessary when using e.g. git which ensures the integrity of the files.